packages/docs/locales/en.json

```diff
@@ -3782,6 +3782,17 @@
         "styles": "Styles",
         "stylesDescription": "* All typography classes now have `va` prefix. For instance: link renamed to `va-link`, `text--right` renamed to `va-text-right`",
         "typography_1": "* `display-1` has been renamed to `va-h1`, same for 2 to 6"
+      },
+      "1_6": {
+        "components": "Components",
+        "navbar": "* `center` slot was replaced with default one;\n * default (`center`) slot is now visible on mobiles.",
+        "innerLoading": "CSS variables names were changed.",
+        "scrollContainer": "`size` prop acceptable values names were changed.",
+        "image": "Component was reworked. Suggest to check all places of usage.",
+        "extensions": "Extensions",
+        "aggrid_0": "Ag Grid",
+        "aggrid_1": "Removed SCSS mixins for Vuestic UI theme customization.",
+        "tailwind": "Added {'@vuestic/tailwind'} package for config syncing between Vuestic UI and Tailwind CSS."
```

Comment:
"Deprecated grid. Use Tailwind instead. "

It is migration guide, not release log.

Reply:
I'll change, but imo migration guide should contain something like `do it, or you'll face issues in this version`. Unlikely grid system in 1.6.0.